### PR TITLE
update(Injury): Allow staff & police inspect cmd

### PR DESCRIPTION
**Pull request by Savage912-Official**
Added the ability for staff to use the inspect command rather than just police.

Requested by: @Khajit71

### CODE REVIEW

**Comment by minipunch** [comment]
Why @Khajit71  ?

**Comment by Khajit71** [comment]
Because plenty of times we get /helps from ppl that their injuries got glitched and they need heal, and we need to double check with their injury log if injury's match. That, or if we need to check someone's injuries sometimes we don't have cop staff on @minipunch

**Comment by Savage912-Official** [comment]
@minipunch

File: resources/usa_injury/sv_injury.lua [hunk 1]
```diff
@@ -79,13 +79,19 @@ end, {
...
 	local targetSource = tonumber(args[2])
-	if targetSource and GetPlayerName(targetSource) then
-		TriggerEvent('injuries:getPlayerInjuries', targetSource, _source)
-	else
-		TriggerClientEvent("injuries:inspectNearestPed", _source, _source)
+	local user = exports["essentialmode"]:getPlayerFromId(source)
+	local char = exports["usa-characters"]:GetCharacter(source)
+	local job = char.get("job")
+	local group = user.getGroup()
+	if job == "corrections" or job == "sheriff" or job == "cop" or job == "ems" or group == "mod" or group == "admin" or group == "superadmin" or group == "owner" then
```
Comment (minipunch) [inline resources/usa_injury/sv_injury.lua:89]: Looks like we lost 'doctor' from here ?

**Review by minipunch — CHANGES_REQUESTED** [review]

File: resources/usa_injury/sv_injury.lua (hunk 1 above)
Comment (Savage912-Official) [reply resources/usa_injury/sv_injury.lua]: Fixed

**Review by minipunch — APPROVED** [review]

**Pull request merged**